cedar-policy-core/Cargo.toml

```diff
@@ -42,6 +42,7 @@ chrono = { version = "0.4.38", optional = true, default-features = false}
 
 # protobuf dependency
 prost = { version = "0.13", optional = true }
+educe = "0.6.0"
 
```

Comment:
is the "protobuf dependency" section the correct place to put `educe`?

Reply:
It was added automatically by cargo. I will adjust it.